x-pack/legacy/plugins/graph/public/components/field_manager/field_manager.tsx

```diff
@@ -35,6 +36,7 @@ export function FieldManagerComponent(props: {
   deselectField: (fieldName: string) => void;
   pickerOpen: boolean;
   setPickerOpen: (open: boolean) => void;
+  store?: GraphStore; // only for testing purpose
```

Comment:
I think it would be clearer if we also put https://github.com/airbnb/enzyme/issues/2176#issuecomment-532361526 here as a comment